### ISSUE TITLE
Names in English in the Open Muni API 

### ISSUE THREAD

**Issue by kobiluria** [kobiluria/MoneyMap#12]
@pwalsh 

My system is after the first step and i wanted to make it a little bit more friendly. 

Was the choice for the name of the entities in Hebrew an api decision or can you get to the name_en info with an api call ?

Thanks.

**Comment by pwalsh** [comment]
@kobiluria yes it was a conscious decision. I'm upgrading the locale API for the codebase in general, and the web API will mirror the web app, where, a subdomain activates a translation. The previous solution was not scalable for scenarios we support, where for example an instance of Open Budgets could have 10 - 20 languages in practice, and this would affect the size of each JSON payload.

There is always a default language ('he' in our current case), and that is served on the default domain ('ext' or 'www' in the current case). 

When the refactor goes live, you'll access translations like this:

www.example.com/api/SOMETHING
en.example.com/api/SOMETHING
ar.example.com/api/SOMETHING
